plugins/ssr/ssrparser.ml4

```diff
@@ -2050,7 +2051,7 @@ END
 (** Rewrite arguments *)
 
 let pr_option f = function None -> mt() | Some x -> f x
-let pr_pattern_squarep= pr_option (fun r -> str "[" ++ pr_rpattern r ++ str "]")
+let pr_pattern_squarep= pr_option (fun r -> str "[" ++ Ssrmatching_plugin.Ssrmatching.pr_rpattern r ++ str "]")
```

Comment:
There is an open up there, I guess this is line is spurious

Reply:
Do you want me to fix this?